levelkdev/dxswap-dapp
# Show rewards list when not connected

# Summary

Fixes #653 

*Show all the rewards pair instead of connect button when you are not connected*
![Screenshot 2022-03-07 at 17 39 24](https://user-images.githubusercontent.com/33226956/157077727-93e6e3ac-5a8a-4854-86f5-ee1082063f07.png)


  # To Test

1. Disconnect wallet
2. Go to rewards section
2. Check if rewards list appears even when wallet is not connected
